src/main/kotlin/com/example/cms/domains/preparedcontents/models/entities/PreparedContent.kt

```diff
@@ -33,6 +33,8 @@ class PreparedContent : BaseEntity() {
     @Column(name = "status", nullable = false)
     lateinit var status: ContentStatuses
 
+    @Column(name = "css_style_classes")
```

Comment:
change it to `css_classes`
style is redundant

Reply:
Done